src/components/gltf-model-plus.js

```diff
@@ -376,9 +338,38 @@ export async function loadGLTF(src, contentType, preferredTechnique, onProgress,
     Object.keys(fileMap).forEach(URL.revokeObjectURL);
   }
 
+  gltf.scene.animations = gltf.animations;
+
   return gltf;
 }
 
+export async function loadModel(src, contentType = null, useCache = false, jsonPreprocessor = null) {
```

Comment:
I assume this just got shuffled around. Be extra careful when merging to make sure nothing gets lost.